packages/protocol/test/automata-attestation/AutomataDcapV3AttestationTest.t.sol

```diff
@@ -0,0 +1,110 @@
+
+    function setUp() public {
+        // uint256 fork = vm.createFork(rpcUrl);
+        // vm.selectFork(fork);
+
+        // pinned September 23rd, 2023, 0221 UTC
+        // comment this line out if you are replacing sampleQuote with your own
+        // this line is needed to bypass expiry reverts for stale quotes
+        vm.warp(1_695_435_682);
+
+        vm.deal(admin, 100 ether);
+
+        vm.startPrank(admin);
+        p256Verifier = new P256Verifier();
+        sigVerifyLib = new SigVerifyLib(address(p256Verifier));
+        pemCertChainLib = new PEMCertChainLib();
+        attestation = new AutomataDcapV3Attestation(address(sigVerifyLib), address(pemCertChainLib));
+        attestation.setMrEnclave(mrEnclave, true);
+        attestation.setMrSigner(mrSigner, true);
+
+        string memory tcbInfoJson = vm.readFile(string.concat(vm.projectRoot(), tcbInfoPath));
+        string memory enclaveIdJson = vm.readFile(string.concat(vm.projectRoot(), idPath));
+
+        string memory fmspc = "00606a000000";
+        (bool tcbParsedSuccess, TCBInfoStruct.TCBInfo memory parsedTcbInfo) =
+            parseTcbInfoJson(tcbInfoJson);
+        require(tcbParsedSuccess, "tcb parsed failed");
+        attestation.configureTcbInfoJson(fmspc, parsedTcbInfo);
+
+        (bool qeIdParsedSuccess, EnclaveIdStruct.EnclaveId memory parsedEnclaveId) =
+            parseEnclaveIdentityJson(enclaveIdJson);
+        require(qeIdParsedSuccess, "qeid parsed failed");
+        attestation.configureQeIdentityJson(parsedEnclaveId);
```

Comment:
Same here as before, I think would be nice to share this set up code between the different places.